policyengine_us/parameters/gov/states/ca/cdss/tanf/child_care/eligibility/resource_limit/property/base.yaml

```
@@ -0,0 +1,12 @@
+description: California limits the CalWORKs Child Care to filers with property value below this amount in a household with no aged or disabled members.
```

And elsewhere; `filers` typically means tax filing units
```suggestion
description: California limits CalWORKs Child Care to households with property value below this amount in a household with no aged or disabled members.
```